modal/_container_entrypoint.py

```diff
@@ -829,9 +841,14 @@ def breakpoint_wrapper():
 
         with container_io_manager.handle_user_exception():
             finalized_functions = service.get_finalized_functions(function_def, container_io_manager)
-
         # Execute the function.
         try:
+            for finalized_function in finalized_functions.values():
+                if finalized_function.lifespan_manager:
+                    event_loop.create_task(finalized_function.lifespan_manager.background_task())
+                    call_lifecycle_functions(
```

Comment:
Now that I think about it, it might be a bit cleaner to use the lower level

```py
with container_io_manager.handle_user_exception():
    event_loop.run(finalized_function.lifespan_manager.lifespan_startup())
```

Since `call_lifecycle_functions` does some other wonky things that we might want to refactor at some point